packages/core/util/types/index.ts

```diff
@@ -42,6 +42,7 @@ export interface AbstractSessionModel extends AbstractViewContainer {
   assemblies: AnyConfigurationModel[]
   selection?: unknown
   duplicateCurrentSession(): void
+  pushSnackbarMessage(message: string): void
```

Comment:
Do we think all our views will support `pushSnackbarMessage`? And are messages like that always going to go to a snackbar? In JB embedded, maybe it will report errors some other way. Might be worth us talking about this a bit in grooming.

This doesn't necessarily block merging this though.

Reply:
That's a good point. Maybe we have a generic `alert` action on the session? In jb-web that will trigger a snackbar, but other products could display it differently.

I'd actually like the generic `alert` to handle levels of alerts, too, like "success," "info," "warning," and "error." Then in jb-web we could have customize the snackbars like [this](https://material-ui.com/components/snackbars/#customized-snackbars).